.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: ['7.4', '8.0', '8.1']
+        php-version: ['7.4', '8.2']
```

Comment:
Why did you remove 8.0 and 8.1? They can in theory behave different. As long as they are supported I would like to keep them. Same for 7.4 as long as the library technically works with it it should stay even if the php version is no longer supported.

Reply:
There is no point in also checking the ones in between.